src/main/java/com/powsybl/openloadflow/ac/nr/LineSearchStateVectorRescaler.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.ac.nr;
+
+import com.powsybl.openloadflow.ac.equations.AcEquationType;
+import com.powsybl.openloadflow.ac.equations.AcVariableType;
+import com.powsybl.openloadflow.equations.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class LineSearchStateVectorRescaler implements StateVectorRescaler {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LineSearchStateVectorRescaler.class);
+
+    private static final int MAX_ITERATION = 10;
+    private static final int STEP_FOLD = 4;
```

Comment:
mu = 0.25 seems very conservative to me. Maybe something smoother like 0.75 (STEP_FOLD = 4/3) is more appropriate.

Reply:
But in that case we should increase the number of max iterations?

Reply:
0.75^10 = 0.056... so I think it is sufficient, don't you agree ?

Reply:
Yes, let's try with 0.75